client/auth/test/login.jsx

```diff
@@ -12,16 +12,32 @@ import useFakeDom from 'test/helpers/use-fake-dom';
 import useMockery from 'test/helpers/use-mockery';
 
 describe( 'LoginTest', function() {
-	let Login, loginStub, page, React, ReactDom, ReactInjection, TestUtils;
+	let Login, loginStub, pushAuthLoginStub, page, React, ReactDom, ReactInjection, TestUtils;
+
+	const POLL_INTERVAL = 6000;
```

Comment:
it's a good question whether to redeclare this here or import from the file where it's defined